History/HistoryEntry.h

```diff
@@ -61,9 +61,86 @@ struct HistoryInlineRef
     bool        mIsRemote   : 1;
     bool        mIsTag      : 1;
     bool        mIsStash    : 1;
+    bool        mIsDetached : 1;
+
+    bool operator ==(const HistoryInlineRef& other) const
```

Comment:
What is the purpose of these operators? It looks like they're actually just implementing default behaviour.